.github/workflows/run-integration-test-mbed.yaml

```diff
@@ -0,0 +1,104 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Mbed OS - Run integration test
+
+# Controls when the action will run. 
+on:
+  workflow_run:
+    workflows: ["Build example - Mbed OS"]
+    types: 
+      - completed
+
+permissions:
+  actions: none
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  test:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+
+    # Container definition that will run the test
+    container:
+        image: ubuntu:latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+        # Fetch depth 0 to get all history and be able to check mergepoint for bloat report
+        with:
+            fetch-depth: 0
+            submodules: false
+      - name: Install dependencies
+        run: |
+          apt-get update
+          apt-get install -y lavacli wget jq curl
+          wget https://github.com/mikefarah/yq/releases/download/v4.9.3/yq_linux_amd64 -O /usr/bin/yq 
+          chmod +x /usr/bin/yq
+      - name: Binary artifact suffix
+        id: outsuffix
+        uses: haya14busa/action-cond@v1.0.0
+        with:
+            cond: ${{ github.event.pull_request.number == '' }}
+            if_true: "${{ github.sha }}"
+            if_false: "pull-${{ github.event.pull_request.number }}"
+
+      - name: Try to get last succesfull |run_id| from build_cr.yaml workflow 
+        run: |
+              echo "CHIPTOOL_CR_RUN_ID=$(\
+              curl --silent \
+              --request GET \
+              --url 'https://api.github.com/repos/${{ github.repository }}/actions/workflows/build_cr.yaml/runs?per_page=1&status=success' \
+              --header 'authorization: Bearer ${{ secrets.GITHUB_TOKEN }}' \
+              --header 'content-type: application/json' | \
+              jq '.workflow_runs[0].id' \
+              )" \
+              >> $GITHUB_ENV
+      - name: Verify correctness of requested |run_id|
+        run: |
+            if [[ "$CHIPTOOL_CR_RUN_ID" == "null" ]]; then
+            echo "::error::Can't obtain last succesfull run_id from build_cr.yaml workflow"
+            exit 1
```

Comment:
To get this work we have to use bash as a default shell for all jobs in this workflow.
I've already fixed this in #188